corehq/apps/smsbillables/interface.py

```diff
@@ -129,6 +140,7 @@ def _format_billables(self, sms_billables):
         return [
             [
                 sms_billable.date_sent,
+                BillingAccount.get_account_by_domain(sms_billable.domain).name,
```

Comment:
I'm wondering if this might be prohibitively expensive (time-wise) to look up separately for each row. Since the domain-to-account mapping is fairly stable (i.e. it does not change frequently), it's a good candidate for caching, which should be straightforward to add. I think just decorating the function definition of `get_account_by_domain` with something like `@quickcache(['domain'])` would do it

Reply:
@dannyroberts Added!

Reply:
Looks like my suggestion didn't work as simply as I'd hoped. Apparently this is because `get_account_by_domain` is written to accept _either_ a domain name (which is what I thought and would work fine with quickcache) _or_ a domain object (which is sometimes used, and which doesn't play nicely with quickcache).

I think an easy workaround that still lets us cache here is to pull `BillingAccount.get_account_by_domain(sms_billable.domain).name` out into a function that takes a domain name (_name_ only, never object) and returns an account name. And then put `@quickcache(['domain'])` on _that_. Alternatively you can just revert the caching change altogether. Given that it's just one page of results at a time it has to do this for (right?) I may have been over-optimizing on what might not even be the slowest part of the workflow. Given that this is an internal UI, as long as we test it for reasonable speed (like it doesn't time out) after deploying this, that's fine.